test/specs/ScrollingSpec.ts

```diff
@@ -15,15 +15,17 @@ export namespace PageLocator {
 describeCommon('Scroll', () => {
   it('scrollIntoView ', () => {
     BrowserUtils.waitForDisplayed(PageLocator.HEADER);
-    assert.isFalse($(PageLocator.CAT_IMAGE).isDisplayedInViewport());
-    BrowserUtils.scrollIntoView(PageLocator.CAT_IMAGE);
-    assert.isTrue($(PageLocator.CAT_IMAGE).isDisplayedInViewport());
+    assert.isFalse($(PageLocator.LAST_TOP_LINK).isDisplayedInViewport());
```

Comment:
Suggest to add test that scroll to an element that already in viewport (first element)

Reply:
done